DirectX11/HackerContext.cpp

```diff
@@ -2647,31 +2647,34 @@ STDMETHODIMP_(void) HackerContext::IASetIndexBuffer(THIS_
 	/* [annotation] */
 	__in  UINT Offset)
 {
-	FrameAnalysisLog("IASetIndexBuffer(pIndexBuffer:0x%p, Format:%u, Offset:%u)\n",
-			pIndexBuffer, Format, Offset);
-	FrameAnalysisLogResourceHash(pIndexBuffer);
-
 	mOrigContext->IASetIndexBuffer(pIndexBuffer, Format, Offset);
 
-	// When hunting, save this as a visited index buffer to cycle through.
-	if (pIndexBuffer && !G->mDataBuffers.empty() && G->hunting == HUNTING_MODE_ENABLED) {
+	// Save the current index buffer hash for indexbufferfilter in ShaderOverride.
+	if (pIndexBuffer && !G->mDataBuffers.empty()) {
```

Comment:
For this one, does it also make sense to look for an active shader or buffer override?

The .find Map operations are expensive, and profiling has shown them to be our biggest performance problem, so as a general rule I'm trying to avoid lookups in scenarios where nothing is to change.  

See for example the SetShader template.